backend/spotihue/hue.py

```diff
@@ -24,7 +24,7 @@ def white_light(self) -> bool:
         """
```

Comment:
What do you think about declaring a HueLight.rgb property here, under HueLight.white_light?
something like:
`    @property
    def rgb(self) -> Tuple[int, int, int]:
        x, y = self.xy
        r = int(x * 255)
        g = int(y * 255)
        b = int((1 - x - y) * 255)
        return r, g, b`

OR
`    @property
    def rgb(self) -> Tuple[int, int, int]:
        x, y = self.xy
        return self._convert_to_rgb(x, y)
`
where the _convert_to_rgb method is declared privately on the HueLight class? or maybe as a standalone function at the top of the hue.py file (whatever you prefer)?

Then, we could have:
`
for light in self.hue_bridge.reachable_lights:
    lights_rgb.append({"light_name": light.name, "light_rgb": light.rgb})
`

Reply:
OK the code formatting is screwed up but you get it 😅